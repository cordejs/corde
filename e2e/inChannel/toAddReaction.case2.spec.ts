import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("tests should fail", async () => {
  const command = Utils.buildCommandWithConfigPath("inChannel", "bot_case2.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(mockProcess).toBeCalledWith(1);
  expect(stdout).toMatchSnapshot();
});
