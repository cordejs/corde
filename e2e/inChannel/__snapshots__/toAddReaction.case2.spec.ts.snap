// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests should fail 1`] = `
"node test --config ./e2e/corde.config.ts --files ./e2e/inChannel/__cordeTest__/bot_case2.spec.ts

 FAIL   e2e\\\\inChannel\\\\__cordeTest__\\\\bot_case2.spec.ts   5.289s
   x <empty test name>   5.279s
    
    expected: testing bot to send a message
    received: no message was sent

    at toReturn Object.toReturn (lib\\\\src\\\\expect\\\\index.js:33:58)
    at e2e\\\\inChannel\\\\__cordeTest__\\\\bot_case2.spec.ts:6:78
    at _internalTest (lib\\\\src\\\\closures\\\\test.js:29:13)

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       5.291s
"
`;
